src/collection.ts

```diff
@@ -1151,6 +1199,33 @@ export class Collection<
     )
   }
 
+  /**
+   * Retrieves one document from the KV store according to the given options.
+   *
+   * If no options are given, the first document in the collection is retreived.
+   * 
+   * @example
+   * ```ts
+   * // Get the first user
+   * const { result } = await db.users.getOne()
+   *
+   * // Get the first user with username that starts with "a"
+   * const { result } = await db.users.getOne({
+   *   filter: doc => doc.value.username.startsWith("a")
+   * })
+   * ```
+   *
+   * @param options - List options, optional.
+   * @returns A promise that resovles to the retreived document
+   */
+  async getOne(options?: ListOptions<Document<TOutput>>) {
+    return await this.handleMany({
+      ...options,
+      limit: null,
```

Comment:
Instead of overriding (and consequently disabling) the limit option, I suggest leaving it at the default behaviour. This would enable use cases where the user wants to perform the operation on a subset of the collection. That should also play well with the pagination options from other methods.